components/Pledge/Merci.js

```diff
@@ -20,8 +20,8 @@ import { EMAIL_CONTACT, ONBOARDING_PACKAGES } from '../../lib/constants'
 
 import {
   A,
-  Interaction,
-  RawHtml,
+  H1,
+  P,
```

Comment:
`H1` and `P` are [legacy editorial elements from the crowdfunding](https://styleguide.republik.ch/typography#legacy-editorial-content-styles). Why switch from `Interaction`? Generally we use `Interaction` for selling (e.g. störer, angebot seite etc.)

Reply:
IMO we should eventually change account page to be sans serif / interaction as well.